src/Sylius/Behat/Page/Admin/Product/UpdateSimpleProductPage.php

```diff
@@ -158,6 +158,15 @@ public function isMainTaxonChosen(string $taxonName): bool
         return $taxonName === $this->getDocument()->find('css', '.search > .text')->getText();
     }
 
+    public function isTaxonChosen(string $taxonName): bool
+    {
+        $productTaxonsElement = $this->getElement('product_taxons');
+
+        $taxonName = str_replace('-', '_', $taxonName);
+
+        return strtolower($taxonName) === strtolower($productTaxonsElement->getValue());
```

Comment:
Not sure if we need this anymore
```suggestion
        return$taxonName === $productTaxonsElement->getValue();
```